test/Riok.Mapperly.Tests/Mapping/UnsafeAccessorTest.cs

```diff
@@ -0,0 +1,334 @@
+﻿using Riok.Mapperly.Abstractions;
+using Riok.Mapperly.Diagnostics;
+
+namespace Riok.Mapperly.Tests.Mapping;
+
+[UsesVerify]
+public class UnsafeAccessorTest
```

Comment:
Add a test without unsafe accessors but with the member visibility filter (`IncludedMembers`).

Reply:
I'm not sure what you mean here. Do you want me to add a test where the .NET version doesn't support `UnsafeAccessor`. I'm not sure how to do this. Should it be an integration test?

Reply:
Currently only `MemberVisibility.All` and `MemberVisibility.AllAccessible` are tested, also add tests for at least another value (eg. `Public` and check whether only public members are mapped).